apps/web/app/pages/sbtc-rewards/sbtc-rewards.tsx

```diff
@@ -40,7 +42,7 @@ const sbtcEnroll = {
   id: 'basic',
   logo: <SbtcLogo size="32px" />,
   title: 'Basic sBTC rewards',
-  description: 'Hold sBTC in your wallet to passively earn more sBTC, as it compounds over time.',
+  description: formatPostPrompt(content.posts["sbtc-rewards-basic"]?.Prompt || ''),
```

Comment:
Fallback to the original text?

Reply:
I spoke with Kyran about this briefly on Friday. I believe @kyranjamie and I were both thinking that if content is loaded at _build_ time (rather than runtime) then fallbacks shouldn't be needed. 

My main concern with fallbacks in the code (for copy and content generally) is that they can easily get out of date with the source of truth, which may be more dangerous than simply throwing a build-time error that prompts source correction. But perhaps worth a discussion?